items.json

```diff
@@ -4429,9 +4429,9 @@
     "name": "Highland Spiked Club",
     "culture": "Battania",
     "type": "OneHandedWeapon",
-    "price": 107,
-    "weight": 1.42,
-    "tier": 0.355965823,
+    "price": 2376,
+    "weight": 1.3,
+    "tier": 5.14793873,

```

Comment:
we should have on high tier spiked mace. Maybe this one because it does not look that bad 
![image](https://user-images.githubusercontent.com/47090987/202036136-dc58370d-67d5-4b8b-b396-acf7af961bca.png)

Reply:
The morning star is what I see as the high tier mace: 
![image](https://user-images.githubusercontent.com/118079593/202041666-5e616250-6530-471e-abc8-b61e58625c87.png)

Reply:
High tier should be above tier 9

Reply:
Just committed the change for the morning star